cli.js

```diff
@@ -13,61 +18,47 @@ Usage:
   parsec-switcher -l | --list
   parsec-switcher -h | --help
   parsec-switcher setup <parsecdLocation>
-  
   parsec-switcher --version
 
 Options:
   -h --help     Show this screen.
   --version     Show version.
 `;
 
-const options = docopt(doc, { version: '0.0.1' });
+const options = docopt(doc, { version: packageJSON.version });
 
-try {
-    if(options['setup']){
-        console.log("Running setup")
-        runSetup(options['<parsecdLocation>'])
-    }
-
-        if(setupRequired()){
-            if(!autoSetup()){
-                throw new Error("Setup required, run parsec-switcher setup \"Your Parsecd.exe location\", this will nuke your parsec-switcher data and accounts")
-            }
-        }
+if(!flags.parsecDataLocationFound){
+    console.log("Parsec not installed")
+}
+else if(!flags.parsecdFound){
+    console.log("Parsecd not found, run \"parsec-switcher setup <parsecdLocation>\" ")
 
+} else {
+    if (options['-a'] || options['--add']){
```

Comment:
if/else doesn't look good for validation, instead wrap everything in main funciton and use if/return

Reply:
done